src/test/scala/com.lightbend.paradox/apidoc/ApidocDirectiveSpec.scala

```diff
@@ -86,6 +92,26 @@ class ApidocDirectiveSpec extends MarkdownBaseSpec {
       )
   }
 
+  it should "allow linking to a typed class that is also present in classic" in {
+    markdown("@apidoc[typed.*Replicator$]") shouldEqual
```

Comment:
I am not too sure whether the regexp is the best syntax here. Especially since the `$` at the end does not mean regexp end-of-line here.

Should we instead have a designated syntax for skipping one or more number of package names? It could still be star, and this would become `@apidoc[typed.*.Replicator$]`.

Reply:
And then `@apidoc[typed.*\.ActorContext]` from https://github.com/akka/akka/pull/26666/files#r271677048 could become `@apidoc[typed.*.ActorContext]`.

Reply:
Good suggestion! A little less powerful but clearer to read and write.

Also we can always later introduce regexes with `@apidoc[/foo/]` syntax if the need arises

Reply:
Yep, good idea for putting regexp between `/`.